testing/screenshots/index.js

```diff
@@ -46,25 +47,53 @@ function getOptions() {
 }
 
 (async function(){
+    process.argv.forEach((val, index) => {
+        console.log(`${index}: ${val}`);
+      });
+
```

Comment:
Is this just for debugging or is it meant to stick around?

Reply:
I kept it in so we can see what the actions pass if anything.